src/main/java/com/uber/cadence/internal/compatibility/proto/serviceclient/GrpcServiceStubs.java

```diff
@@ -124,9 +123,6 @@ final class GrpcServiceStubs implements IGrpcServiceStubs {
             MetadataUtils.newAttachHeadersInterceptor(headers),
             newOpenTelemetryInterceptor(),
             newOpenTracingInterceptor(options.getTracer()));
-    if (log.isTraceEnabled()) {
-      interceptedChannel = ClientInterceptors.intercept(interceptedChannel, tracingInterceptor);
-    }
```

Comment:
I would probably just keep it. For unit test, we can just cover the lines without asserting as it's for debugging purposes.

Reply:
Also seeing this pattern in multiple other entities. For example ActivityPollTask

Reply:
Sounds good, readded with test coverage.